coding/python.md

```diff
@@ -7,13 +7,23 @@ This document outlines the rules for writing Python code across all our Projects
 
 ## Code style and formatting
 
-All code must conform to [PEP 8](https://pep8.org/) and be formatted with [black](https://github.com/ambv/black) using the option `--line-length 79`.
+We use [Flake8](https://flake8.pycqa.org/en/latest/) to conform to [PEP 8](https://pep8.org/). It will suggest you to remove unsued variables, remove unused imports, enforce more strict whitespace practices and calculate your code complexity. E.g. `flake8 webapp tests`.
```

Comment:
>, remove unused imports, 

I'm not aware of `flake8` removing anything on its own - AFAIK it will only checks, it doesn't format. Or maybe you meant that it will "suggest you to ... remove unused imports", in which case I think the wording could be tidied up.

I was willing to believe you that it complains about unused variables, but in my testing this doesn't seem to be the case.

I am also not aware of it having an capability to calculate code complexity (and I'm not aware of PEP8 talking about code complexity either). It would be great to have something to calculate code complexity though.

I'm don't quite understand what's meant by "enforce strict whitespace practices". I think this could be made clearer.

How about simply:

```suggestion
We use [Flake8](https://flake8.pycqa.org/en/latest/) to check our code for compliance with [PEP 8](https://pep8.org/), e.g. with `flake8 webapp tests`. See [the list of Flake8 violations](https://flake8.pycqa.org/en/latest/user/error-codes.html) for more information.
```